src/main/javascript/crypto/src/eas/EasTicketAttestation.ts

```diff
@@ -225,12 +228,15 @@ export class EasTicketAttestation extends AttestableObject implements Attestable
 		return this.decodedData;
 	}
 
-	getAttestationField(fieldName: string){
+	getAttestationField(fieldName: string, suppressException = false){
```

Comment:
Hi, @micwallace , could you explain me reason why do we need to allow some empty fields?

Reply:
I added the idea of having a default key for all eventIds, this was mainly for testing purposes.
```
base64senderPublicKeys: {
   "":"publickey pem..."
}
```

When a key for a specific isn't found, it will try to use the default key for validation. But this parameter is now redundant as I am using getAttestationData instead: 
src/main/javascript/crypto/src/eas/EasTicketAttestation.ts:445

Just pushed an update to remove it.

Reply:
@oleggrib I added the concept of a default key that can be used when a key for the specific devconId isn't set explicitly. Or for attestations that don't use eventId or devconId. This was initially adding for testing purposes.
```
"base64senderPublicKeys": {
   "": "PEM public key..."
}
```

I'm using getAttestationData() for this now so have removed the redundant suppressException parameter.